main.go

```diff
@@ -80,9 +66,14 @@ func main() {
 	flag.BoolVar(&tls, "tls", true, "Enable TLS (with self-signed certificate)")
 	flag.Parse()
 
+	mdlw := middleware.New(middleware.Config{
```

Comment:
will this middleware emit a request count with status?

Reply:
correct.